Assets/MixedRealityToolkit.SDK/Experimental/Features/Boundary/Scripts/BoundarySystemManager.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Boundary
+{
+    /// <summary>
+    /// Service manager supporting running the boundary system, without requring the MixedRealityToolkit object.
```

Comment:
>requring [](start = 72, length = 8)

requiring

Reply:
I think this misspelling is copied into the rest of the system managers, so you may want to do a Ctrl+F across all files.

---
In reply to: [301753191](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5233#discussion_r301753191) [](ancestors = 301753191)

Reply:
yeah copy/paste typo :)

Reply:
fixed